pipelines/common/src/main/java/org/openmrs/analytics/DwhFiles.java

```diff
@@ -42,54 +55,36 @@ public class DwhFiles {
 
   private static final Logger log = LoggerFactory.getLogger(DwhFiles.class);
 
-  private static final String TIMESTAMP_FILE = "timestamp.txt";
+  public static final String TIMESTAMP_FILE = "timestamp.txt";
```

Comment:
Any particular reason you have made this and the next one `public`? If not, it is probably better not to expose/use these outside this class, for now.

Reply:
I had made them public to use it in the test classes. I have reverted to private for now.